### PR TITLE
fix issue #8

### PR DESCRIPTION
**Pull request by xbelt**
Fix for issue #8 
Later I will improve the code further but for now it works

### CODE REVIEW

File: Source/Platforms/Windows/Dialogs/CrashReportForm.cs [hunk 1]
```diff
@@ -58,5 +59,24 @@ public sealed partial class CrashReportForm : Form {
             DialogResult = DialogResult.OK;
             Close();
         }
+
+        private void CrashReportForm_Load(object sender, EventArgs e) {
+            _originalHeight = Height;
+            ErrorContentsBox.Text = Client.Configuration.Resolver.GetJsonSerializer().Serialize(Event);
```
Comment (niemyjski) [inline Source/Platforms/Windows/Dialogs/CrashReportForm.cs]: I wonder if we should put this in a try catch.. Just in case serialization fails.

**Comment by xbelt** [comment]
Added the try catch and improved the form.
The screenshots will be available in the discussion about issue #8.

**Comment by niemyjski** [comment]
Thanks! We've updated issue #8.

**Comment by niemyjski** [comment]
I'm going to close this as it also needs to support wpf. Please feel free to open a pull request in the future with support for both (Ideally, we'd love if this opened a new dialog or layer ontop of this dialog with the content).

**Pull request closed without merging**